README.rst

```diff
@@ -1,20 +1,26 @@
-****************
+
+******************************
 django CMS Version Locking
-****************
+******************************
 
-## Explanation
-The version-locking addon is intended to modify the way djangocms-versioning works.  
-* The primary change is that it locks a version to its author when a draft is created. 
-* That version becomes automatically unlocked again once it is published. 
-* The lock prevents editing of the file by anyone other than the author.
-* Locks can be removed by a user with the correct permission
-* Unlocking an item sends a notification to the author to which it was locked.
-* Manually unlocking a version does not lock it to the unlocking user, nor does it change the author.
-* The Version admin view for each content-type registered with Version-locking will be edited to add in lock icons / buttons UI to the Actions column.
+Explanation
+------------
+The version-locking addon is intended to modify the way djangocms-versioning works in the following way: 
 
+ - The primary change is that it locks a version to its author when a draft is created. 
+ - That version becomes automatically unlocked again once it is published. 
+ - The lock prevents editing of the file by anyone other than the author.
+ - Locks can be removed by a user with the correct permission
```

Comment:
This README might not be the place for it, but I do think this permission should be specified precisely somewhere. This is rather important knowledge for anybody who might be creating their own roles/groups etc. and having to search the code for it would be annoying.